tests/Doctrine/Tests/Common/Annotations/AbstractReaderTest.php

```diff
@@ -408,6 +408,30 @@ public function testInvalidAnnotationButIgnored()
         self::assertEmpty($reader->getPropertyAnnotations($class->getProperty('foo')));
     }
 
+    public function testGloballyIgnoredAnnotationNotIgnored() : void
+    {
+        $reader = $this->getReader();
+        $class  = new \ReflectionClass(Fixtures\ClassDDC1660::class);
+
+        $active = true;
```

Comment:
Another way to disable this autoloader would be using a `ReflectionProperty` like in `AnnotationRegistryTest`, not sure if it is any better.